test/app-luatest/msgpack_test.lua

```diff
@@ -445,3 +446,91 @@ g.test_object_misc = function()
     t.assert_not(msgpack.is_object(it))
     t.assert_not(msgpack.is_object({mp}))
 end
+
+local test_array = {true, 1, '2', -2, 3.0, {1, 2, 3}, {a = 1, b = 2, c = 3}, 'decode', 'iterator', 'get'}
+local test_map = {a = nil, [1] = true, ['1'] = 1, b = '2', [-2] = 3.0, ['-2'] = 4, [2.5] = 5, [true] = 6,
+                  c = {a = 7, b = 8, c = 9}, decode = 10, iterator = 11, get = 12}
```

Comment:
Nit: Please keep lines within 80 characters in Lua files. I added a checkpatch check for this.

Reply:
This is questionable: according to luacheck [documentation](https://luacheck.readthedocs.io/en/stable/warnings.html?highlight=max%20line%20length#line-length-limits), the default line length limit is 120.

Anyways, you should rather patch [.luacheckrc](https://github.com/tarantool/tarantool/blob/master/.luacheckrc) than checkpatch.

Reply:
> This is questionable: according to luacheck [documentation](https://luacheck.readthedocs.io/en/stable/warnings.html?highlight=max%20line%20length#line-length-limits), the default line length limit is 120.

We follow the 80 character limit in most our Lua files. 120 characters don't fit in my screen:

![image](https://user-images.githubusercontent.com/1114511/205269554-11cdc7c5-ff0a-4210-99bd-4c9d9502b671.png)

> Anyways, you should rather patch [.luacheckrc](https://github.com/tarantool/tarantool/blob/master/.luacheckrc) than checkpatch.

I don't want to update luacheckrc now, because it's applied to all Lua files and we do have a few black sheep in the flock (tests mostly). I don't want to patch them now. I just want to prevent new files with long lines from appearing. Also, there may be exceptions when we do want to ignore this rule (e.g. long tables). Checkpatch failure may be ignored while luacheck must always pass, because it's run on the master.

Reply:
Fixed.

Regarding, the left `LONG_LINE_STRING` errors, please see my [comment](https://github.com/tarantool/tarantool/pull/7994#issuecomment-1334231161).

Reply:
You don't even need to break the line here:

```Lua
    t.assert_error_msg_content_equals(
        "bad argument #1 to '?' (msgpack.object expected, got number)",
```

Anyway, when a line is too long, you can break it by punctuation, e.g.:

```Lua
    t.assert_error_msg_content_equals("bad argument #1 to '?' " ..
                                      "(msgpack.object expected, got number)",
```

A dev will hardly try to grep the whole message. `grep "bad argument"` is more likely.

Reply:
I agree with your points, fixed: split the lines as you suggest.